website/src/react-native/showcase.js

```diff
@@ -205,6 +205,12 @@ var featured = [
     linkPlayStore: 'https://play.google.com/store/apps/details?id=me.ele.crowdsource',
     author: 'Eleme',
   },
+  {
+    name: '饿小闲',
+    icon: 'https://xiaoxian.ele.me/download/images/share_page/ic_launcher.png',
+    link: 'https://xiaoxian.ele.me/download/?td_channelid=RNshowcase',
+    author: 'Eleme',
+  },
 ];
```

Comment:
@rlog Can you please move this down to the "Apps" section, since the "Featured" section is reserved. Thanks!